.travis.yml

```diff
@@ -0,0 +1,64 @@
+---
+language: node_js
+node_js:
+  # we recommend testing addons with the same minimum supported node version as Ember CLI
+  # so that your addon works for all apps
+  - "10"
```

Comment:
Since we are going to major, we might as well drop Node 10 too (since it's already EOL'ed)

Reply:
good point